.buildkite/filebeat/pipieline.yml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+name: "beats-auditbeat"
```

Comment:
typo in the filename and `name` is not matching the name of the folder.

Reply:
Oh. It was a leftover pipeline.yml. Deleted it